dashboard/app/helpers/notes_helper.rb

```diff
@@ -3,13 +3,24 @@ module NotesHelper
 
   def get_slides_by_video_key(video_key)
     slides_key = "slides.#{video_key}"
-    slides = try_t(slides_key)
-    return nil unless slides
+    slides = {}
+    1.step.each do |n|
```

Comment:
maybe I'm misunderstanding this, but it looks like this (and the slides_english version below) will only grab data from the first slide.

Reply:
[`Numeric#step`](https://ruby-doc.org/core-2.5.0/Numeric.html#method-i-step) invokes the block with a sequence of numbers (incremented by 1 by default), until the `break` is hit.